src/main/kotlin/com/terraformation/backend/tracking/api/ObservationsController.kt

```diff
@@ -149,6 +149,25 @@ class ObservationsController(
     )
   }
 
+  @ApiResponseSimpleSuccess
+  @Operation(summary = "Stores the results of a completed observation of a plot.")
+  @PostMapping("/ad-hoc/monitoring")
+  fun completeAdHocPlotMonitoringObservation(
+      @RequestBody payload: CreateAdHocPlotMonitoringObservationRequestPayload,
```

Comment:
Nit: The function name (which is used as the OpenAPI endpoint name) uses the word "complete" but the payload names use the word "create."